backend/api/Models/AccessRequestModel.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Pims.Api.Models
+{
+    public class AccessRequestModel : BaseModel
+    {
+        #region Properties
+        public Guid Id { get; }
+        public UserModel User { get; set; }
+        public IEnumerable<AgencyModel> Agencies { get; set; }
+        public bool? IsGranted { get; set; }
+        public IEnumerable<RoleModel> Roles { get; set; }
```

Comment:
Only allow for a single role.  While we support multiple, we will only allow them to select one.